instrumentation/github.com/gin-gonic/gin/otelgin/gintrace.go

```diff
@@ -95,7 +97,7 @@ func Middleware(service string, opts ...Option) gin.HandlerFunc {
 			span.SetAttributes(semconv.HTTPStatusCode(status))
 		}
 		if len(c.Errors) > 0 {
-			span.SetAttributes(attribute.String("gin.errors", c.Errors.String()))
+			span.RecordError(errors.New(c.Errors.String()))
```

Comment:
Shouldn't we call `span.RecordError` for each error inside `c.Errors`?

Reply:
Can it be like this? I'll try.

Reply:
I tried 2 options to see which one would recommend? My decision is Option 2:

- one:

	```go
	if len(c.Errors) > 0 {
		var errs error
		for _, err := range c.Errors {
			errs = errors.Join(errs, err.Err)
		}
		span.RecordError(errs)
	}
	```

- two:

	```go
	if len(c.Errors) > 0 {
		for _, err := range c.Errors {
			span.RecordError(err.Err)
		}
	}
	```

Reply:
One would end up being exactly the same thing as we had before. So two.